### DISCUSSION TITLE
[Hardware] Louder Buzzer Mod.

### DISCUSSION THREAD

**Discussion by 814d3** [joeycastillo/Sensor-Watch#192] [Ideas]
Hi Folks, I just got the idea of having a louder buzzer. Maybe it is also possible to tweak it a bit via software ore maybe there are louder buzzer units to exchange 'em with the stock one. A maybe simple idea, but a web search didn't bring any useful results. Maybe any of you have some ideas.
Greetings

**Comment by joeycastillo** [comment]
This would be an improvement to make on a future Sensor Watch board. The standard buzzer is fine buzzing louder; the issue is we’re only driving it with 3 volts because that’s all the microcontroller can push. 

The standard F-91W connects its buzzer output to a transistor that switches an inductor, which lets it shove a lot more voltage onto the piezo; I haven’t measured it but it could easily be pushing dozens of volts peak to peak. Replicating that circuit should be simple; the challenge is finding room for the additional parts on this already jam-packed board :)

**Reply by 814d3** [reply]
Thanks for your extensive reply. I'm looking forward to the upcoming board versions or any improvements that are yet to come ;-)

**Comment by Jonas-fr** [comment]
There also could also be a lowtech approach where you'd modify the enclosure (the back of the watch) by adding a cavity to make it resonate more. The downside would be that you will maybe lose the water resistance. The upside is that it wouldn't consume more power.

**Comment by 814d3** [comment]
Just a little hint as I explored beep distortions und sometimes very faint beeps or complete dropouts of the buzzer and thought it could be a low battery. But it seemed to be bad contacts of the buzzer. I scratched the contact surface with a screwdriver and the buzzer worked way better with a clearer sound.

**Comment by WesleyBlack** [comment]
Hi, I know this is an older post, but I recently created a watch face that lets you cycle through and trigger all the 87 buzzer notes.

I created it see if I could find a louder note for alarms, and I am happy to report I found one that is louder than my other digital watches. My results won't be the same as yours as I fitted the assembled movement to a different case that had a similarly sized digital movement.

You can download the code from my fork (beeps_face.h, and beeps_face.c) just make sure to add `#include "beeps_face.h"` to the movement_faces.h file. My fork is located here: https://github.com/WesleyBlack/Sensor-Watch/tree/main/movement/watch_faces/demo

Hope this can be of help

**Comment by 814d3** [comment]
Hi, nice idea and thanks for the pull request. 
Could you give a hint, what note you chose in spite of your case?
And as I'm not so into the code, if I want to change the beep note of the timer_face, I just have to alter the BUZZER_NOTE_C8 in the following line?
`static const int8_t _sound_seq_beep[] = {BUZZER_NOTE_C8, 3, BUZZER_NOTE_REST, 3, -2, 2, BUZZER_NOTE_C8, 5, BUZZER_NOTE_REST, 25, 0};`

**Reply by WesleyBlack** [reply]
BUZZER_NOTE_B8 was the loudest one in my case (though it is on the higher pitch side) the next loudest notes were BUZZER_NOTE_A8SHARP_B8FLAT, BUZZER_NOTE_E7, and BUZZER_NOTE_G6.

Yes you would just need to swap out both instances of BUZZER_NOTE_C8  for the new note in the code you provided.
